R/zzz.R

```diff
@@ -134,7 +108,10 @@ chooseOpsMethod.S7_object <- function(x, y, mx, my, cl, reverse) TRUE
 }
 
 .onLoad <- function(...) {
-  ## "S4"   or [in R-devel 2023-07-x]   "object"
+  on_load_define_ops()
```

Comment:
If you like this pattern, I'll do the rest of `.onLoad()` so the bulk on the code lives close to where it's used.

Reply:
I have no preference, both work for me :) 

One minor thought, I originally wrote the code so that, eventually, when we drop support for R 4.3.0, the diff could just be deleting the block of lines (i.e., it was a self-contained block)

Reply:
I think that's a reasonable goal, but the string "4.3.0" now appears in 7 files, so I don't think it's feasible any more.